py3status/modules/README.md

```diff
@@ -295,9 +302,9 @@ Obsolete configuration parameters:
     *(default None)*
 
 Requires:
-  - the `acpi` command line
+  - the `acpi` command line (only if `measurement_mode='acpi'`)
 
-**author** shadowprince, AdamBSteele, maximbaz, 4iar
+**author** shadowprince, AdamBSteele, maximbaz, 4iar, m45t3r
```

Comment:
these changes should be made in the module docstring and the README can be updated automatically by running `py3status docstring update battery_level`

the `acpi` command line __utility__ (only if `measurement_mode='acpi'`)